extensions/guacamole-auth-cas/src/main/java/org/apache/guacamole/auth/cas/ticket/TicketValidationService.java

```diff
@@ -57,9 +57,7 @@
      *     If the ID ticket is not valid, the username claim type is missing, or
      *     guacamole.properties could not be parsed.
      */
-    public String processUsername(String ticket) throws GuacamoleException {
-
-        AttributePrincipal principal = null;
+    public AttributePrincipal validateTicket(String ticket) throws GuacamoleException {
```

Comment:
The documentation for this function states:

> Validates and parses the given ID ticket, returning the username contained therein, ...

This was from when the function returned a `String` and was called `processUsername()`. Now that the function has been reworked to return an `AttributePrincipal` (and renamed to `validateTicket()`), is this documentation still valid?

Reply:
Updated.